Description: Not clear why upstream force -std=c++11
 Current gcc default is c++17 so use that instead.
Author: Mathieu Malaterre <malat@debian.org>
Origin: upstream
Forwarded: https://github.com/libjxl/libjxl/pull/1100
Last-Update: 2021-12-15

Index: libjxl/CMakeLists.txt
===================================================================
--- libjxl.orig/CMakeLists.txt
+++ libjxl/CMakeLists.txt
@@ -224,9 +224,9 @@ find_package(Threads REQUIRED)
 
 # These settings are important to drive check_cxx_source_compiles
 # See CMP0067 (min cmake version is 3.10 anyway)
-set(CMAKE_CXX_STANDARD 11)
+set(CMAKE_CXX_STANDARD 17)
 set(CMAKE_CXX_EXTENSIONS OFF)
-set(CMAKE_CXX_STANDARD_REQUIRED YES)
+set(CMAKE_CXX_STANDARD_REQUIRED NO)
 
 # Atomics
 find_package(Atomics REQUIRED)
